torch/onnx/symbolic_opset9.py

```diff
@@ -2561,8 +2561,8 @@ def prim_tolist(g, input, dim_val, elem_ty_val):
     return input
 
 
-@parse_args('v', 'i')
```

Comment:
@SplitInfinity can you or someone else familiar with ONNX please verify these changes? Any possibility of BC-breaking issues here?

Reply:
cc: @BowenBao

Reply:
Thanks @jbschlosser, @SplitInfinity. 
I think this is safe for BC-breaking. It would be great if you could update the usage of `dtype` in the symbolic function as well to respect your latest change. Here are a few code references for help:
1. Example for parsing dtype value
```python
    def _get_arange_dtype(dtype):
        dtype = sym_help._maybe_get_const(dtype, 'i')
        if sym_help._is_value(dtype):
            dtype = 4  # default to int64
        return dtype
```
from https://github.com/pytorch/pytorch/blob/729d7b5123d9f073ee8833de04b870894f3287a3/torch/onnx/symbolic_opset9.py#L2650
2. Cast final output according to dtype
https://github.com/pytorch/pytorch/blob/729d7b5123d9f073ee8833de04b870894f3287a3/torch/onnx/symbolic_opset9.py#L2660
3. Add to existing test case
https://github.com/pytorch/pytorch/blob/729d7b5123d9f073ee8833de04b870894f3287a3/test/onnx/test_pytorch_onnx_onnxruntime.py#L3257

Reply:
Thanks for the input @BowenBao! @thomasjpfan Can you please make the changes described above?

Reply:
I updated the PR with the suggested changes. I also updated `ort_compare_with_pytorch` in `test_pytorch_onnx_onnxruntime.py` to actually check the dtypes when `check_dtypes=True`.